src/user/user.controller.ts

```diff
@@ -16,6 +16,15 @@ export class UserController {
   create(@Body() createUserDto: CreateUserDto) {
     return this.userService.create(createUserDto);
   } 
+  
+  @Patch(':id')
+  @ApiOperation({ summary: 'Modify user' })  
+  @ApiCreatedResponse({ description: 'User status has been successfully updated' }) 
+  @ApiBadRequestResponse({ description: 'Bad request. Check your input data' }) 
+  update(@Param('id') id: string, @Body() updateUserDto: UpdateUserDto) {
+    // console.log(updateUserDto)
+    return this.userService.update(+id, updateUserDto);
+  }
```

Comment:
Is plus operator necessary?

Reply:
The 'id' parameter is a string; adding the plus operator converts it to a number.